response_operations_ui/templates/collection-exercise.html

```diff
@@ -8,8 +8,21 @@
     <h1 class="jupiter">{{survey.shortName}} {{ce.name}}</h1>
 
     <div class="grid">
-      <div class="grid__col col-5@l"></div>
-      <div class="grid__col col-1@l"></div>
+      <div class="grid__col col-6@l">
+        <section class="ce-section">
+          <h2 class="venus u-pt-m">Collection instruments</h2>
+          <div class="u-mb-m">
+            <form action="" class="form">
+              <p>
+                Choose a collection instrument (CI) to load
+              </p>
+              <label class="upload-file u-vh" for="file">Choose CI file</label>
+              <input type="file" class="" name="ciFile">
+              <button id="btn-load-ci" class="btn btn--primary unready" type="submit">Load CI</button>
+            </form>
+          </div>
+        </section>
+      </div>
 
       <div class="grid__col col-6@l">
```

Comment:
I'm not sure if this is a change that was made or if I made a mistake in the previous PR but there should be an empty grid column size 1 between the left and right and this grid column should be size 5 according to the dropbox html